kernel/src/hil/touch.rs

```diff
@@ -0,0 +1,83 @@
+//! Interface for touch input devices
+
+use crate::ReturnCode;
+
+#[derive(Debug, Copy, Clone)]
+pub enum TouchStatus {
+    Pressed,
+    Released,
+}
+
+#[derive(Debug, Copy, Clone)]
+pub enum GestureEvent {
+    MoveUp,
+    MoveDown,
+    MoveLeft,
+    MoveRight,
```

Comment:
`Move` is maybe ambiguous here (what is moving, the finger, the object being clicked on?) -- perhaps `SwipeUp` etc as "Swipe" is more clearly a gesture?

Reply:
Good point.